### PR TITLE
Added support for FRM303, enhancement 2863

### LINKED ISSUE #2863: Support Flysky FRM303
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

FRM303 is currently not working in any radios, seems the drivers are not properly configured.

FRM303 used the same protocol as the IRM301 in the EL18 with the following information:
1. Radio TX used the PPM pin
2. Radio RX used the heartbeat pin
3. UART bit rate is 1.5M BAUD
4. Signaling is non-inverted (Idle high)

I think supporting this should be easy, just tweaks a bit on the init and config.

### Describe the solution you'd like

Please update the init and config for AFHDS3 ext module to support it properly.

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by richardclli**
Fixes #2863

Summary of changes:

### CODE REVIEW

**Comment by richardclli** [comment]
I have successfully added support FRM303, and it works in my EL18.  This change will not affect the MPM, the MPM is ok as well.

@raphaelcoeffic There are 2 points may need to check:
1. The current FRM303 uses normal TX/RX while the MPM uses inverted TX, I just put the code in the init and deinit of the ext AFHDS3 module. 
2. The ext FRM303 and int IRM301 uses different framing address, not sure if we need to put the selection in the UI as well.

**Comment by richardclli** [comment]
Besides Flysky radio, I think it may not work on TX16S because of the buggy MB.

**Comment by richardclli** [comment]
Rebased after module-api merged.  Need to test again later this week.

File: radio/src/pulses/afhds3.cpp [hunk 1]
```diff
@@ -857,6 +864,7 @@ static void* initModule(uint8_t module)
   params.baudrate = AFHDS3_UART_BAUDRATE;
   params.polarity =
     module == INTERNAL_MODULE ? ETX_Pol_Normal : ETX_Pol_Inverted;
+//    module == ETX_Pol_Normal;
```
Comment (raphaelcoeffic) [inline radio/src/pulses/afhds3.cpp:867]: Please remove this line ;-)

**Comment by raphaelcoeffic** [comment]
@richardclli other than the extra line, I'm good with it, let's have this module supported!

**Comment by richardclli** [comment]
@raphaelcoeffic However, I discovered something bad when testing it.  A crash occurs when using the internal IRM301 alone and did not plug in the FRM303.  Not sure why.  It works ok without my changes.

**Comment by richardclli** [comment]
The crash is finally fixed.  It works now for both int module IRM301 and ext module FRM303.

**Comment by richardclli** [comment]
@pfeerick This should be ready to merge to main.

**Review by pfeerick — APPROVED** [review]

**Comment by jmxp69** [comment]
Just to confirm, when merged, FRM303 works with EL18 only and not TX16s?

**Comment by richardclli** [comment]
Yes, not worked for now, hardware limitations.  Need Flysky to work out another serial connection mode in FRM303, before we can work on the support for other radios.

**Comment by xiaoyigg** [comment]
Is it possible in the future?

**Comment by richardclli** [comment]
Sure will work in the future.

**Pull request merged**